apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -202,13 +215,15 @@ public void multipleReferenceYearsInSingleFileForSameBeneExpectBeneMonthlyPopula
           "Beneficiary has incorrect mbiHash");
       assertEquals(expectedHicn, beneficiaryFromDb.getHicn(), "Beneficiary has incorrect HICN");
     } finally {
-      if (entityManager != null) entityManager.close();
+      if (entityManager != null) {
+        entityManager.close();
+      }
     }
   }
 
   /**
    * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data for an <code>
-   * UPDATE</code> {@link Beneficiary} record that there hasn't been a previous <code>INSERT</code>
+   *  UPDATE</code> {@link Beneficiary} record that there hasn't been a previous <code>INSERT</code>
```

Comment:
Curious if this shouldn't be further indented?

Reply:
This and the others are the same issue anywhere it linebreaks on a bracket it'll add a space. Could replace them all with the inline `{@code x}` style but I havent been doing that thusfar so I'll leave these.